scalableminds/webknossos
# Mutexes finishAndGetNext

### Mailable description of changes:
 - FinishAndGetNext can only be triggered once at a time

### Steps to test:
- Create a bunch of tasks
- Get one task and start tracing
- Add the user script below
- Press `7` repeatedly in short sequence
- No errors should occur and you should get a new task

```js
webknossos.apiReady(3).then(async (api) => { 
  api.utils.registerKeyHandler("7", () => {
    api.tracing.setCommentForNode("unsolved", api.tracing.getActiveNodeId(), 
      api.tracing.getActiveTreeId());
    api.utils.showToast("warning", 'Task marked as unsolved. Moving on to next task...', 3000);
    api.tracing.finishAndGetNextTask();
  });
});
```

### Issues:
- fixes #2448

------
